docs/develop/best-practices/accessibility.md

```diff
@@ -26,9 +26,449 @@ The technology is standards-compliant and performant.
+        <div class="list">
+            <div class="list-item">List item 1</div>
+            <div class="list-item">List item 2</div>
+        </div>
+        <span class="button">Button text</span>
+    </div>
+</div>
+
+<div class="ui fluid container">
+    Footer content
+</div>
+```
+
+### Increase heading levels by one
+Always use the correct level of heading. {==Each page should have one, and only one, `<h1>` heading, followed by increasing levels: `<h2>`,`<h3>`,`<h4>`...==}
+
+Headings should be followed by either the same heading-level or one level up, depending on the content structure. 
+
+Never skip heading levels only for styling purposes. If you want a different size, use styling instead. Semantic UI already has classes you can combine with `.ui .header` like this:
+```html
+<!-- Use styling instead of skipping heading levels -->
+
+<h2 class="ui huge header">Huge Header</h2>
+<h2 class="ui large header">Large Header</h2>
+<h2 class="ui medium header">Medium Header</h2>
+<h2 class="ui small header">Small Header</h2>
+<h2 class="ui tiny header">Tiny Header</h2>
+```
+
+_With Semantic UI React:_
+
+```html
+<Header as="h2" size='huge'>Huge Header</Header>
+<Header as="h2" size='large'>Large Header</Header>
+<Header as="h2" size='medium'>Medium Header</Header>
+<Header as="h2" size='small'>Small Header</Header>
+<Header as="h2" size='tiny'>Tiny Header</Header>
+```
+Note that it's recommended to also be consistent with heading sizes.
+
+✅ __DO__
+```html
+
+<!-- 
+Only one h1 per page, 
+always increase heading level with one or zero 
+-->
+
+<h1>Page title</h1>
+
+<h2>Section 1</h2>
+<h3>Sub heading 1</h3>
+<h3>Sub heading 2</h3>
+
+<h2>Section 2</h2>
+
+<h2>Section 3</h2>
+```
+
+_With Semantic UI React:_
+
+```js
+/* 
+Only one h1 per page,
+always specify your heading level in Semantic UI React,
+increase by one or zero 
+*/
+
+<Header as='h1'>Page title</Header>
+
+<Header as='h2'>Section 1</Header>
+<Header as='h3'>Sub heading 1</Header>
+<Header as='h3'>Sub heading 2</Header>
+
+<Header as='h2'>Section 2</Header>
+
+<Header as='h2'>Section 3</Header>
+```
+
+❌ __DON'T__
+```html
+<!-- 
+Don't skip heading levels or add more 
+than one h1 to the same page 
+-->
+
+<h1>Page title</h1>
+
+<h1>Section 1</h1>
+<h4>Sub heading 1</h4>
+
+<h1>Section 2</h1>
+
+<h3>Section 3</h3>
+```
+
+_With Semantic UI React:_
+
+```js
+// Don't leave out the as-prop
+
+<Header size="huge">Page title</Header>
+
+<Header size="large">Section 1</Header>
+<Header size="medium">Sub heading 1</Header>
+
+<Header size="large">Section 2</Header>
+
+<Header size="large">Section 3</Header>
+```
+
+### Give images alt tags
+All images should have an descriptive alt-tag.
+
+✅ __DO__
+```html
+<!-- Always include the alt-tag -->
+
+<img src="/my-images/image.png" alt="Image description"/>
+```
+
+_With Semantic UI React:_
+
+```js
+// Always include the alt-tag
+
+<Image src="/my-images/image.png" alt="Image description"/>
+```
```

Comment:
If the image is purely decorative, you should hide it from screenreaders with an empty alt tag. 
<Image src="/my-images/curlicue.png" alt="" >